applications/DEM_application/python_scripts/DEM_procedures.py

```diff
@@ -99,7 +99,8 @@ def __init__(self, model_parts_list):
         self.cluster_model_part    = self.Get("ClusterPart")
         self.DEM_inlet_model_part  = self.Get("DEMInletPart")
         self.mapping_model_part    = self.Get("MappingPart")
-        self.contact_model_part    = self.Get("ContactPart")        
+        self.contact_model_part    = self.Get("ContactPart")
+        self.rigid_body_model_part = self.Get("RigidBodyPart")
```

Comment:
What is this modelpart for? The clusters can be in the ClustersModelPart and the Walls in the RigidFacePart. What other Rigid bodies do we have?